docs/api/latest.md

```diff
@@ -0,0 +1,190 @@
+# API Docs - v1.0.0-SNAPSHOT
```

Comment:
Remove this file

Reply:
removed in d1f03a8fee06db4c0e05dabedc5f1af320126d5b